### PR TITLE
Add HotDogStand theme

### PR DESCRIPTION
**Pull request by maciejpedzich**
It's been a long time coming... but at long last... I've ported over the one and only HotDogStand theme to this amazing project. Enjoy!

### CODE REVIEW

File: src/themes/hot-dog-stand.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
+html,
+body {
+	overflow: hidden;
+	height: 100vh;
+	margin-top: 0;
+}
+
+body {
+	background-color: yellow;
+	color: red;
```
Comment (fimion) [inline src/themes/hot-dog-stand.css:10]:
```suggestion
	color: black;
```

Comment (maciejpedzich) [reply src/themes/hot-dog-stand.css:10]: Sure, why not.

**Comment by fimion** [comment]
@all-contributors Please add @maciejpedzich  for content and design!

**Review by fimion — APPROVED** [review]
#
Approved!
Approved

**Comment by maciejpedzich** [comment]
LESSSS GOOOOOO

**Pull request merged**